packages/document/doc-tools-doc/modern.config.ts

```diff
@@ -50,6 +50,10 @@ export default defineConfig({
           label: 'English',
         },
       ],
+      editLink: {
+        docRepoBaseUrl: 'https://github.com/test/blog/',
```

Comment:
This is the test case, not the real repo url

Reply:
Should I delete this paragraph or leave an empty string

Reply:
You should leave a real repo url.

Reply:
What form or value is this real address